bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdMetadata.java

```diff
@@ -86,4 +86,15 @@ public interface ClangdMetadata {
 	 */
 	PreferenceMetadata<String> additionalOptions();
 
+	/**
+	 * Returns the metadata for the "Log to Console" option, must not return <code>null</code>.
+	 *
+	 * @return the metadata for the "Log to Console" option
+	 *
+	 * @see ClangdOptions#logToConsole()
+	 *
+	 * @since 2.2
+	 */
+	PreferenceMetadata<Boolean> logToConsole();
```

Comment:
There needs to be a default to be acceptable by the [CDT policy](https://github.com/eclipse-cdt/cdt/blob/main/POLICY.md#new-default-methods-in-interfaces).

(I didn't compile/format this)

```suggestion
	default PreferenceMetadata<Boolean> logToConsole() { return false; } ;
```